internal/installcfg/installcfg.go

```
@@ -223,9 +226,27 @@ func getBasicInstallConfig(log logrus.FieldLogger, cluster *common.Cluster) *Ins
 			NoProxy:    generateNoProxy(cluster),
 		}
 	}
+
+	if cluster.ImageInfo.MirrorRegistriesConfig != "" {
+		_ = setImageContentSources(log, cluster.ImageInfo.MirrorRegistriesConfig, cfg)
```

Don't ignore errors.